commit 7a465bf0d6f613fa82d0e5c3115facdeaf5e92aa
Author: Anatol Pomozov <anatol.pomozov@gmail.com>
Date:   Fri Aug 19 05:13:29 2016 -0700

    Remove dynamic gem installation. Quarry does not like non-tracked dependencies

diff --git a/ext/mkrf_conf.rb b/ext/mkrf_conf.rb
index 028496a..d38b0bc 100644
--- a/ext/mkrf_conf.rb
+++ b/ext/mkrf_conf.rb
@@ -1,21 +1,3 @@
-require 'rubygems/dependency_installer'
-
-# SEE: http://en.wikibooks.org/wiki/Ruby_Programming/RubyGems#How_to_install_different_versions_of_gems_depending_on_which_version_of_ruby_the_installee_is_using
-
-di = Gem::DependencyInstaller.new
-
-begin
-  if RUBY_VERSION =~ /^1.8/
-    puts "Not installing test-unit. It is included in Ruby #{RUBY_VERSION}"
-  else
-    di.install 'test-unit', '~> 3.2'
-  end
-rescue => e
-  warn "#{$0}: #{e}"
-  
-  exit!
-end
-
 puts "Writing fake Rakefile"
 
 # Write fake Rakefile for rake since Makefile isn't used
